bears/cmake/CMakeLintBear.py

```diff
@@ -21,11 +21,14 @@ class CMakeLintBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
+                         cmake_filter: str=None,
                          cmakelint_config: path=''):
         """
         :param cmakelint_config: The location of the cmakelintrc config file.
         """
         args = ()
         if cmakelint_config:
             args += ('--config=' + cmakelint_config,)
+        if cmake_filter:
+            args += ('--filter=' + cmake_filter,)
```

Comment:
@kriti21, Sorry, I missed the `param` part last time. It will be better to show `param` of `cmake_filter` too. 😉